server/go.mod

```diff
@@ -27,7 +27,6 @@ require (
 	github.com/stretchr/testify v1.7.0
 	github.com/tidwall/gjson v1.7.3 // indirect
 	golang.org/x/crypto v0.0.0-20210322153248-0c34fe9e7dc2
-	golang.org/x/oauth2 v0.0.0-20200107190931-bf48bf16ab8d
```

Comment:
This is actually used. It's used in enterprise repo and since go mod doesn't follow symlinks, it thinks this is unused and removes it. 

We shouldn't remove this.

Reply:
We can revert `go.mod` and `go.sum files`.

Reply:
That means we'll have to be hand editing these files.  I don't recall having to do that for chat server.  I'll revert and raise this next meeting so we can figure out a solution that doesn't require hand editing.

Reply:
Fixed

Reply:
Agree on fixing this t avoid manually editing it.